packages/extension-chakra-storefront/src/components/product-scroller/index.jsx

```diff
@@ -139,11 +139,13 @@ const ProductScroller = forwardRef(
                                     id: 'product_scroller.assistive_msg.scroll_left',
                                     defaultMessage: 'Scroll products left'
                                 })}
-                                icon={<ChevronLeftIcon color="black" />}
                                 borderRadius="full"
-                                colorScheme="whiteAlpha"
```

Comment:
Small ask, do you want to take some time to extract these styling in this component into its own theme file? 

When we first created  this components, there was a rush to deadline so we used in-styling. This component is a pretty defined component. Creating a theme file for it may make it easier to manage in the future